### PR TITLE
Add 2201.5.1 release note

### PR DESCRIPTION
**Pull request by LakshanWeerasinghe**
## Purpose
$subject

## Checklist

- [ ] **Page addition**
  - [x] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.5.1/RELEASE_NOTE.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201-5-1
+    - /downloads/swan-lake-release-notes/2201.5.1/
+    - /downloads/swan-lake-release-notes/2201-5-1-swan-lake/
+    - /downloads/swan-lake-release-notes/2201-5-1-swan-lake
+---
+
+## Overview of Ballerina Swan Lake 2201.5.1
+
+<em>Swan Lake 2201.5.1 is the first patch release of Ballerina 2201.5.0 (Swan Lake Update 5) and it includes a new set of bug fixes to the language, tooling, and standard library.</em>
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.1/RELEASE_NOTE.md:15]:
```suggestion
<em>Swan Lake 2201.5.1 is the first patch release of Ballerina 2201.5.0 (Swan Lake Update 5) and it includes a new set of bug fixes to the language and standard library.</em>
```
Check if we have tooling updates going in this release and remove.

File: downloads/swan-lake-release-notes/swan-lake-2201.5.1/RELEASE_NOTE.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Install Ballerina
+
+If you have not installed Ballerina, then, download the [installers](/downloads/#swanlake) to install.
+
+## Language updates
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.5.1 (Swan Lake)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.5.1+label%3AType%2FBug+is%3Aclosed).
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.1/RELEASE_NOTE.md:32]:
```suggestion
To view bug fixes, see the [GitHub milestone for 2201.5.1 (Swan Lake)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.5.1+label%3AType%2FBug+is%3Aclosed).
```
This link is empty.

File: downloads/swan-lake-release-notes/swan-lake-2201.5.1/RELEASE_NOTE.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Standard library updates
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.5.1 (Swan Lake)](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aissue+milestone%3A2201.5.1+label%3AType%2FBug+is%3Aclosed+).
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.1/RELEASE_NOTE.md:38]:
```suggestion
To view bug fixes, see the [GitHub milestone for 2201.5.1 (Swan Lake)](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aissue+milestone%3A2201.5.1+label%3AType%2FBug+is%3Aclosed+).
```
This link is empty.

**Review by praneesha — CHANGES_REQUESTED** [review]

**Review by sm1990 — APPROVED** [review]

File: downloads/swan-lake-release-notes/swan-lake-2201.5.1/RELEASE_NOTE.md (hunk 3 above)
Comment (LakshanWeerasinghe) [reply downloads/swan-lake-release-notes/swan-lake-2201.5.1/RELEASE_NOTE.md:38]: will remove later if needed.

File: downloads/swan-lake-release-notes/swan-lake-2201.5.1/RELEASE_NOTE.md (hunk 2 above)
Comment (LakshanWeerasinghe) [reply downloads/swan-lake-release-notes/swan-lake-2201.5.1/RELEASE_NOTE.md:32]: will remove later if needed.

File: downloads/swan-lake-release-notes/swan-lake-2201.5.1/RELEASE_NOTE.md (hunk 1 above)
Comment (LakshanWeerasinghe) [reply downloads/swan-lake-release-notes/swan-lake-2201.5.1/RELEASE_NOTE.md:15]: will remove later if needed.

**Review by praneesha — APPROVED** [review]

**Pull request merged**